packages/Stylers/src/includes.js

```diff
@@ -47,10 +47,40 @@ export const visuallyHidden = css`
   overflow: hidden;
   padding: 0 !important;
   position: absolute !important;
+  white-space: nowrap;
```

Comment:
This improves the way screen readers will read content in visually hidden elements.

Reference: https://medium.com/@jessebeach/beware-smushed-off-screen-accessible-text-5952a4c2cbfe#.2g9wpybl9